library/marathon.py

```diff
@@ -18,13 +18,17 @@
       - Desired state of the marathon application: - present: application is created if it doesn't exist, but it is not updated if the json changed - absent: application is destroyed - updated: application is updated if the json has differences with the existing configuration - test: outputs the diff between the submitted application and the one that is running (if any) - get: outputs the running application
       choices: ["present", "absent", "updated", "test", "get"]
   app_json:
-    required: true
+    required: false
     description:
-      - The json file describing the marathon application. It is required for all the state choices, except 'get' (where an empty string is OK)
+      - The JSON file describing the marathon application.  Either this or app is required for all the state choices, except 'get' and 'absent' (where not supplying it is OK).
   app_id:
     required: false
     description:
-      - The id of the marathon application to 'get' (currently only used with state 'get')
+      - The id of the marathon application to 'get'.  Used when state is 'get' or 'absent'.
+  app:
+    required: false
+    description:
+      - The JSON content, as a string containing valid JSON, describing the marathon application.  Either this or app_json is required for all the state choices, except 'get' and 'absent' (where not supplying it is OK).
```

Comment:
actually, after all the latest changes (also in our ansible manage role),
IMO we should *only* use the `app` rather than the 'app_json' param (that should be renamed to 'app_file' btw) 
that originally we used to get a nice 'ansible --diff' output (gat got replaced by the more generic diff strategy in new 'test' state)
opinion @vincepii !?